db/system_keyspace.cc

```diff
@@ -3657,9 +3657,14 @@ future<service::topology> system_keyspace::load_topology_state() {
 
         if (some_row.has("transition_state")) {
             ret.tstate = service::transition_state_from_string(some_row.get_as<sstring>("transition_state"));
-        } else if (!ret.transition_nodes.empty()) {
-            on_internal_error(slogger,
-                "load_topology_state: topology not in transition state but transition nodes are present");
+        } else {
+            for (auto& [id, rs]: ret.transition_nodes) {
```

Comment:
Can we use `std::all_of` here?

Reply:
I want to print the ID of the node that's in the wrong state.
I changed it to `find_if`